build.gradle.kts

```diff
@@ -422,9 +460,54 @@ subprojects {
         }
     }
 
+    plugins.withId("me.champeau.gradle.japicmp") {
+        afterEvaluate {
+            tasks {
+                val jApiCmp by registering(JapicmpTask::class) {
+                    dependsOn("jar")
+                    // the japicmp "old" version is either the user-specified one, or the latest release.
+                    val userRequestedBase = project.properties["apiBaseVersion"] as String?
+                    val baselineVersion: String = userRequestedBase ?: latestReleasedVersion
```

Comment:
I've never seen this lazy pattern in any other language so far - that's pretty sweet